source/manual/give-a-content-designer-access-to-github.html.md

```diff
@@ -4,7 +4,7 @@ title: Give a content designer access to GitHub
 parent: "/manual.html"
 layout: manual_layout
 section: Accounts
-last_reviewed_on: 2019-07-16
+last_reviewed_on: 2020-03-10
```

Comment:
I was just looking at this page, do you think you could make these links/emails clickable?

![Screenshot 2020-03-10 at 13 16 09](https://user-images.githubusercontent.com/5111927/76315811-590e0780-62d1-11ea-9ba7-41f9aa8baf9e.png)